src/core/math.d

```diff
@@ -76,7 +77,19 @@ real sin(real x) @safe pure nothrow;       /* intrinsic */
  * indeterminate.
  */
 version (LDC)
-    alias rndtol = stdc.llroundl;
+{
+    // use a fake-pure C llround[l] (might actually touch C errno)
```

Comment:
Now necessary because the previous workaround was to make all of `core.stdc.math` pure (!).